exporter/opentelemetry-exporter-otlp-proto-grpc/src/opentelemetry/exporter/otlp/proto/grpc/log_exporter/__init__.py

```diff
@@ -0,0 +1,188 @@
+# Copyright The OpenTelemetry Authors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Optional, Sequence
+from grpc import ChannelCredentials, Compression
+from opentelemetry.exporter.otlp.proto.grpc.exporter import (
+    OTLPExporterMixin,
+    _translate_key_values,
+    get_resource_data,
+    _translate_value,
+)
+from opentelemetry.proto.collector.logs.v1.logs_service_pb2 import (
+    ExportLogsServiceRequest,
+)
+from opentelemetry.proto.collector.logs.v1.logs_service_pb2_grpc import (
+    LogsServiceStub,
+)
+from opentelemetry.proto.common.v1.common_pb2 import InstrumentationLibrary
+from opentelemetry.proto.logs.v1.logs_pb2 import (
+    InstrumentationLibraryLogs,
+    ResourceLogs,
+    SeverityNumber,
+)
+from opentelemetry.proto.logs.v1.logs_pb2 import LogRecord as PB2LogRecord
+from opentelemetry.sdk.logs import LogRecord as SDKLogRecord
+from opentelemetry.sdk.logs import LogData
+from opentelemetry.sdk.logs.export import LogExporter, LogExportResult
+
+
+class OTLPLogExporter(
+    LogExporter,
+    OTLPExporterMixin[SDKLogRecord, ExportLogsServiceRequest, LogExportResult],
+):
+
+    _result = LogExportResult
+    _stub = LogsServiceStub
+
+    def __init__(
+        self,
+        endpoint: Optional[str] = None,
+        insecure: Optional[bool] = None,
+        credentials: Optional[ChannelCredentials] = None,
+        headers: Optional[Sequence] = None,
+        timeout: Optional[int] = None,
+        compression: Optional[Compression] = None,
+    ):
+        super().__init__(
```

Comment:
is additional code needed here to handle `OTEL_EXPORTER_OTLP_LOGS_*` env variables? Something like this: https://github.com/open-telemetry/opentelemetry-python/blob/191a55ce0478822cb054c48495a4ed227a4579d8/exporter/opentelemetry-exporter-otlp-proto-grpc/src/opentelemetry/exporter/otlp/proto/grpc/trace_exporter/__init__.py#L87-L104

Reply:
I initially added this but since they are not defined anywhere (not even experimental/reserved) I reverted the env changes.